docs/components/card/Example.vue

```diff
@@ -7,90 +7,117 @@
-          </div>
+        <w-card class="full-width">
+          <article class="p-16">
+            <h2 class="h4 font-normal">
+              <p><w-clickable @click="console.log('clicked')" class="text-left">Title</w-clickable></p>
+            </h2>
+          </article>
         </w-card>
-        <w-card :selected="selected">
-          <img class="h-128 w-full object-cover" src="https://source.unsplash.com/random/400x404" />
-          <p class="absolute top-12 left-12 bg-aqua-200 text-aqua-900 p-4 rounded-4 text-12">Ukens bolig</p>
-          <div class="p-16">
-            <p class="text-12 ">DNB Eiendom</p>
-            <p><w-clickable @click="selected = !selected" class="text-left">Stilfull og gjennomgående 3-roms m/balkong. Oppusset i 2019. Inkl. bl.a. vv/fyring.</w-clickable></p>
-            <p class="text-14  mb-4">Bøgata 25C, 0655 Oslo</p>
-            <p class="font-bold my-8">52 m<span style="font-size: 10px; vertical-align: super;">2</span> Totalpris: 4 869 039 kr</p>
-            <p class="text-14  mb-0">Eier (Selveier) <span class="">•</span> Leilighet <span class="">•</span> 2 soverom</p>
-          </div>
+        <w-card class="full-width">
+          <article class="p-16">
+            <h2 class="h4 font-normal">
+              <p><w-clickable @click="console.log('clicked')" class="text-left">Title</w-clickable></p>
+            </h2>
+          </article>
         </w-card>
       </div>
-    </div>
+
+
+      <h3 class="h4">Card containing multiple clickable areas</h3>
+      <div class="grid grid-cols-3 gap-24 mb-24">
+        <w-card class="full-width">
+          <article class="p-16">
+            <h2 class="h4 font-normal">
+              <p><w-clickable @click="console.log('clicked')" class="text-left">Title</w-clickable></p>
+            </h2>
+            <a href="#" class="relative z-10">Interactive #2</a>
+          </article>
+        </w-card>
+        <w-card class="full-width">
+          <article class="p-16">
+            <h2 class="h4 font-normal">
+              <p><w-clickable @click="console.log('clicked')" class="text-left">Title</w-clickable></p>
+            </h2>
+            <a href="#" class="relative z-10">Interactive #2</a>
+          </article>
+        </w-card>
+        <w-card class="full-width">
+          <article class="p-16">
+            <h2 class="h4 font-normal">
+              <p><w-clickable @click="console.log('clicked')" class="text-left">Title</w-clickable></p>
+            </h2>
+            <a href="#" class="relative z-10">Interactive #2</a>
+          </article>
+        </w-card>
+      </div>
+
+
+    <p class="pb-32">The card component is ment for things like the result list items. You are free to use any means of layout and positioning within. The card itself needs a interactive Item for the card, the w-clickable is nice for this. If that is supplied with a href it renders as a link, otherwise it renders as a button.</p>
+
```

Comment:
Nice clean-up! Just three minor things: `ment` should be `meant`, and change the **a** to an **an** in this sentence:`The card itself needs a interactive Item` -> `The card itself needs an interactive Item` and in this sentence: `If that is supplied with a href` -> `If that is supplied with an href`. :)